src/build.d

```diff
@@ -29,6 +29,11 @@ __gshared string[string] env;
 __gshared string[][string] flags;
 __gshared typeof(sourceFiles()) sources;
 
+immutable rootDeps = [
```

Comment:
Please add some DDoc to these to help the next person who has to maintain this code.

Reply:
done